docs/HowToFAQ/Errors-Parser.md

```diff
@@ -639,7 +684,25 @@ longer accepted. Use `function` for compiled, non-ghost functions and
 `ghost function` for non-compiled, ghost functions, and similarly for predicates.
 See [the documentation here](../DafnyRef/DafnyRef#sec-function-syntax).
 
-## **Error: there is no such thing as a 'ghost _what_ method'**
+## **Error: there is no such thing as a 'ghost _what_ method'** {#p_no_ghost_function_method}
```

Comment:
While we are there, it's not very search friendly to have "ghost what method" in text. Could you make it explicit into "ghost function method" and "ghost predicate method" ?

Reply:
Done -- though note that in general the entries in the markdown text correspond 1-1 with error messages in the code -- and those error messages typically have lots of variable content.